extensions/http-curl/tests/unit/InvokeHTTPTests.cpp

```diff
@@ -238,14 +238,14 @@ TEST_CASE("HTTPTestsPostNoResourceClaim", "[httptest1]") {
   auto records = plan->getProvenanceRecords();
   std::shared_ptr<core::FlowFile> record = plan->getCurrentFlowFile();
   REQUIRE(record == nullptr);
-  REQUIRE(records.size() == 0);
+  REQUIRE(records.empty());
```

Comment:
I feel like these transformations in `REQUIRE` are not advantageous, as in the former case we get an error like "error: 3 != 0", while in this new case we get "<...> is false"

Reply:
I don't think we really care about the number of events in this test. The real question here is that if got any events or not. So if we break it and get any events here the count doesn't really help with the debugging IMO.